apps/backend/prisma/schema.prisma

```diff
@@ -57,3 +57,19 @@ model CodemodVersion {
   createdAt                DateTime @default(now())
   updatedAt                DateTime @default(now()) @updatedAt
 }
+
+model TokenMetadata {
+  pepperedAccessTokenHashDigest String @id @map("pathd") @db.VarChar(27)
+  backendInitializationVector   String @map("biv") @db.VarChar(22)
+  encryptedUserId               String @map("euid") @db.VarChar(64)
+  createdAt                     BigInt @map("ca")
+  expiresAt                     BigInt @map("ea")
+  claims                        BigInt @map("c")
+  signature                     String @map("s") @db.VarChar(43)
+}
+
+model TokenRevocation {
+  pepperedAccessTokenHashDigest String @id @map("pathd") @db.VarChar(27)
+  revokedAt                     BigInt @map("r")
```

Comment:
```suggestion
  revokedAt                     BigInt @map("ra")
```

Reply:
actually, I keep the same names as we have in [this](https://www.notion.so/codemod/Platform-Authentication-35e8ce29274d48daadce2b890efac1d7) docs. I'll change it soon maybe, we just want right now to make new backend with Prisma works like it was with Sequelize

Reply:
See references in:
https://github.com/codemod-com/codemod/blob/eac40684c326ab711eb2f7b5e7aadbe78e5cbf47/apps/backend/src/db/buildTokenRevocationsRepository.ts#L64
https://github.com/codemod-com/codemod/blob/eac40684c326ab711eb2f7b5e7aadbe78e5cbf47/apps/backend/src/db/buildTokenMetadataRepository.ts#L82